code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -27,12 +27,11 @@
 		var/list/meancritters = list() // list of possible hostile mobs
 		var/list/nicecritters = list() // and possible friendly mobs
 		for(var/T in typesof(/mob/living/simple_animal))
-			var/mob/living/simple_animal/M = new T
-			switch(M.gold_core_spawnable)
+			switch(initial(T:gold_core_spawnable))

```

Comment:
Well, it doesn't -have- to use ```:```, I'm p.sure you can typecast the path to a variable of itself
```var/mob/living/simple_animal/M = T ```switch(initial(M.gold_core_spawnable)``` which, since you know it should always be a simple animal, you should be fine to do.

Source, I did it here: https://github.com/tgstation/-tg-station/blob/master/code/game/gamemodes/changeling/changeling.dm#L86